src/index.js

```diff
@@ -4,13 +4,17 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter as Router } from "react-router-dom";
+import DataContextProvider from "./context/DataContextProvider";
+
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <Router>
-      <App />
+      <DataContextProvider>
```

Comment:
you can use <filterProvider/>  name for filter context instead of using '<DataContextProiver/>'